### PR TITLE
Ben+Gen from NYPL post

### PR DESCRIPTION
**Pull request by ablwr**
Work in progress -- pending one more paragraph in draft from Ben, and at least one image. But opening PR for visibility.

### CODE REVIEW

File: _posts/2017-09-07-interview-with-ben-turkus-genevieve-havemeyer-king-nypl.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+**What does your media ingest process look like? Does your media ingest process include any tests (manual or automated) on the incoming content? If so, what are the goals of those tests?**  
+
+**(GHK)** I should start by mentioning that our unit primarily manages reformatting for audio and moving image (AMI) research collections; born-digital records, still images, and other collections are managed by our Archives Unit and Digital Imaging Unit colleagues (some of whom also use MediaConch!).
+
+Before AMI deliverables reach NYPL’s Media Ingest system, they are reviewed and tested using a combination of custom scripts, proprietary and open-source tools, and manual content inspection. We check fixity, technical specification conformance, metadata validity, signal quality, and adherence to our preservation policies. Our suite of tools includes bagit.py, JSON schema, ajv, MediaConch, mediainfo, QCTools, Wavelab, and human eyes and ears. There are a lot of goals in running these checks, but namely we strive to:
```
Comment (JeromeMartinez) [inline _posts/2017-09-07-interview-with-ben-turkus-genevieve-havemeyer-king-nypl.md]: MediaInfo (I like M and I in uppercase :) )

Comment (ablwr) [reply _posts/2017-09-07-interview-with-ben-turkus-genevieve-havemeyer-king-nypl.md]: Ah, I should have caught that!

File: _posts/2017-09-07-interview-with-ben-turkus-genevieve-havemeyer-king-nypl.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+**(GHK)** Yeah, right. Hi, I’m Genevieve Havemeyer-King. I’ve been the Media Preservation Assistant for the NYPL’s  Preservation of Audio and Moving Image Unit (PAMI) for about a year. As part of the initiative to preserve the Library’s at-risk audiovisual research collections, my primary role so far has been to assist with coordination of mass-digitization for magnetic and optical media. This includes (but is not limited to) refining and documenting our technical specifications, implementing a robust quality control workflow for our high volume of deliverables, tracking digitization progress, and maintaining inventory of our vendor shipments. As the first of several gatekeepers for NYPL’s media ingest system and digital repository, I also assist Ben with migrating and tracking assets on their journey towards long-term digital preservation.
```
Comment (dericed) [inline _posts/2017-09-07-interview-with-ben-turkus-genevieve-havemeyer-king-nypl.md]: some double spacing in the middle of sentences.

**Comment by ablwr** [comment]
Removing WIP -- This is ready to be reviewed and deployed @JeromeMartinez @GuillaumeRoques

**Comment by ablwr** [comment]
@GuillaumeRoques is the best !

**Comment by GuillaumeRoques** [comment]
It's deployed on prod : https://mediaarea.net/MediaConch/2017/09/06/interview-with-ben-turkus-genevieve-havemeyer-king-nypl/

**Pull request merged**